package.json

```diff
@@ -35,6 +35,13 @@
     "7zip-bin-win",
     "7zip-bin-linux"
   ],
+  "concourse": {
+    "dependencies": [
+      "libudev-dev",
+      "libusb-1.0-0-dev",
+      "libyaml-dev"
+    ]
```

Comment:
How'd this handle different dependencies on different platforms?

Reply:
Good point. Divided the object per operating system.